### PR TITLE
add Feudal Tactics

### PR DESCRIPTION
**Pull request by Sesu8642**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: de.sesu8642.feudaltactics.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+app-id: de.sesu8642.feudaltactics
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions: [org.freedesktop.Sdk.Extension.openjdk17]
+command: feudaltactics
+finish-args:
+  - --socket=x11
```
Comment (hfiguiere) [inline de.sesu8642.feudaltactics.yaml:8]:
```suggestion
  - --socket=x11
  - --share=ipc
```

it required for X11 performance.

File: de.sesu8642.feudaltactics.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
+app-id: de.sesu8642.feudaltactics
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions: [org.freedesktop.Sdk.Extension.openjdk17]
```
Comment (hfiguiere) [inline de.sesu8642.feudaltactics.yaml:5]:
```suggestion
sdk-extensions: 
  - org.freedesktop.Sdk.Extension.openjdk17
```

let's write clean YAML

File: de.sesu8642.feudaltactics.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+command: feudaltactics
+finish-args:
+  - --socket=x11
+  - --device=dri
+  - --env=PATH=/app/bin:/app/jre/bin:/usr/bin
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands: [/usr/lib/sdk/openjdk17/install.sh]
```
Comment (hfiguiere) [inline de.sesu8642.feudaltactics.yaml:14]:
```suggestion
    build-commands: 
      - /usr/lib/sdk/openjdk17/install.sh
```

File: de.sesu8642.feudaltactics.yaml [hunk 4]
```diff
@@ -0,0 +1,40 @@
+app-id: de.sesu8642.feudaltactics
```
[the remark this answers is not in the document]
Comment (Sesu8642) [reply de.sesu8642.feudaltactics.yaml:1]: Yes, I do own the domain.

**Comment by hfiguiere** [comment]
bot, build de.sesu8642.feudaltactics

**Comment by Sesu8642** [comment]
@hfiguiere is it ok to have the icon, desktop and appstream files in here? I can also put them in the upstream repo but I'm not sure if it makes sense. Right now, the release jar is sufficient for the build. With the metadata files in the upstream repo, I guess the sources would need to be downloaded as well.

**Comment by hfiguiere** [comment]
We prefer they be upstream for a future release. But it's ok to have them here in the mean time / if needed.

Note that the lack of icon or desktop file for a Linux application is a bug. AppStream is relatively new in comparison. None of this is specific to flatpak, except the requirement.

**Pull request closed without merging**